python/rmm/_lib/memory_resource.pxd

```diff
@@ -112,3 +112,7 @@ cdef class HybridMemoryResource(MemoryResource):
 
 cdef class LoggingResourceAdaptor(MemoryResource):
     cpdef flush(self)
+
+cpdef get_default_resource_type()
+
+cpdef is_initialized()
```

Comment:
Is this needed?

Reply:
The function is declared as `cpdef`, which means it can be accessed from Cython and Python. So a user could either use `import` as usual or they could use `cimport` to link other Cython code against it. The latter requires it be listed in this file. I just added it here since it was being listed previously. To save review time I fixed it in this PR, though arguably it should be in its own PR as it is not strictly related.